docs/kyma/docs/031-gs-local-installation.md

```diff
@@ -16,19 +16,17 @@ To run Kyma locally, clone this Git repository to your local machine and downloa
 - [Helm](https://github.com/kubernetes/helm) 2.8.2
 - [jq](https://stedolan.github.io/jq/)
 - [Hyperkit driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-driver) - Mac only
-- [Virtualbox](https://www.virtualbox.org/) - Linux or Windows
-- Hyper-V - Windows
+- [VirtualBox](https://www.virtualbox.org/) - Linux or Windows
```

Comment:
I'm updating prereqs in https://github.com/kyma-project/kyma/pull/137
so no need to do that

Reply:
ok